.github/workflows/build.yml

```diff
@@ -14,10 +14,14 @@ jobs:
       - name: Checkout Git repository
         uses: actions/checkout@v2.3.4
 
-      - name: Setup Node
+      - name: Read .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
+
+      - name: Setup Node (uses version from .nvmrc)
         uses: actions/setup-node@v2.1.5
         with:
-          node-version: 14
+          node-version: ${{ steps.nvm.outputs.NVMRC }}
```

Comment:
Just a minor point, but I feel NVMRC_VERSION would be a better name since the file itself is named `.nvmrc`

Reply:
The `.nvmrc` file only holds one value - it's a number version of Node to use... So sort of IMO is clear when defining such as variable also... 😏